BTrees/_base.py

```diff
@@ -1269,7 +1269,7 @@ def iteritems(self, min=_marker, max=_marker,
 
     def byValue(self, min):
         return reversed(
-                sorted((v, k) for (k, v) in self.iteritems() if v >= min))
+                sorted(((v, k) for (k, v) in self.iteritems() if v >= min)))
```

Comment:
I'm going to leave this the way it was.  Because a) Not sure what the right answer is, and I don't want to have to add special comparison at the C level.